src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs

```diff
@@ -288,6 +289,15 @@ public CertCompareResultType CompareCertInfo(string packageId)
             return (CertCompareResultType)compareResult;
         }
 
+        /// <summary>
+        /// The allowed packages and required privileges information.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public IReadOnlyDictionary<string, List<string>> AllowedPackagesAndPrivileges
```

Comment:
❌ DO NOT use ArrayList or List<T> in public APIs.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections

```suggestion
        public IReadOnlyDictionary<string, IEnumerable<string>> AllowedPackagesAndPrivileges
```

Reply:
I changed to use IEnumerable<string> instead.